pkg/k8s/apis/cilium.io/v2alpha1/lbipam_types.go

```diff
@@ -69,11 +69,21 @@ type CiliumLoadBalancerIPPoolSpec struct {
 	//
 	// +kubebuilder:validation:Optional
 	ServiceSelector *slimv1.LabelSelector `json:"serviceSelector"`
-	// CiliumLoadBalancerIPPoolCIDRBlock is a list of CIDRs comprising this IP Pool
+	// AllowFirstLastIPs, if set to `yes`` means that the first and last IPs of each CIDR will be allocatable.
+	// If `no` or undefined, these IPs will be reserved.
```

Comment:
nit: According to `lb-ipam.rst`, LB-IPAM will ignore this field when specifying a /31-/32 for IPv4 and /131-/132 for IPv6. Not a bad idea to state this in the godocs here.